readme.md

```diff
@@ -54,16 +75,16 @@ $ npm install --global strip-css-comments
 ```
 $ strip-css-comments --help
 
-  Usage
-    $ strip-css-comments <input-file> > <output-file>
-    $ strip-css-comments < <input-string>
+	Usage
+		$ strip-css-comments <input-file> > <output-file>
+		$ strip-css-comments < <input-string>
 
-  Option
-    -a, --all  Strip all comments including `/*!`
+	Option
+		-a, --all  Strip all comments including `/*!`
 
-  Example
-    $ strip-css-comments src/app.css > dist/app.css
-    $ strip-css-comments < src/app.css --all
+	Example
+		$ strip-css-comments src/app.css > dist/app.css
+		$ strip-css-comments < src/app.css --all
```

Comment:
Not sure. I didn't edit them directly. Perhaps something to do with space to tab conversion. I'm using the editorconfig plugin so I'm surprised to have formatting issues. I went through the readme again and made a few other formatting corrections.